src/main/kotlin/no/nav/familie/ef/sak/vilkår/dto/VilkårDto.kt

```diff
@@ -24,7 +25,7 @@ data class VilkårGrunnlagDto(
     val adresseopplysninger: AdresseopplysningerDto?,
     val dokumentasjon: DokumentasjonFraSøknadDto?,
     val harAvsluttetArbeidsforhold: Boolean?,
-    val harKontantstøttePerioder: Boolean?,
+    val kontantstøttePerioder: List<KontantstøttePeriode>?,
```

Comment:
```suggestion
    val kontantstøttePerioder: List<KontantstøttePeriode>,
```

Trenger denne å være nullable nå?

Reply:
Nei, den kan være non-nullable nå. Endrer